portfolio/src/main/webapp/script.js

```diff
@@ -21,8 +21,8 @@ function expandImage(imgs) {
 
 function getReviews() {
   fetch('/data').then(response => response.json()).then((reviews) => {
-    for(var i = 0; i < reviews.length; i++) {
-      document.getElementById('reviews-container').innerText += reviews[i] + '\n';
+    for(var review of reviews) {
```

Comment:
nbd but I think `let` is slightly more correct here, instead of var.

Reply:
very true. I will change